feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/TabularFormSessionPageFragment.kt

```diff
@@ -40,6 +40,10 @@ class TabularFormSessionPageFragment : DaggerFragment() {
         InjectedViewModelProviders.of(requireActivity()).get(sessionPagesStoreProvider)
     }
 
+    private val args: TabularFormSessionPagesFragmentArgs by lazy {
```

Comment:
すみません。この方法なのですが、Fragmentのインスタンスは再利用されることがあるので、
バグの原因になることがありそうです。

下記をご参照。
https://www.slideshare.net/RecruitLifestyle/kotlin-87339759
P.69

我々からの指摘なのに、すみません 🙇

Reply:
例えば `lateinit` として定義していただき、onActivityCreatedで毎回初期化 が良さそうです。

Reply:
English version:

So sorry :bow:
The method of initializing properties using lazy in Fragment is not good,
Because instance of Fragment may be reused 😇 

What do you think of using lateinit instead?